grails-app/domain/com/careerfly/geo/Address.groovy

```diff
@@ -0,0 +1,16 @@
+package com.careerfly.geo
+
+class Address {
```

Comment:
Read this https://gist.github.com/sagrawal14/713a86d52ff862babe42e6e1a5e4140c#file-groovyjavapractices-groovy-L21